Formula/graphviz.rb

```diff
@@ -1,9 +1,14 @@
 class Graphviz < Formula
   desc "Graph visualization software from AT&T and Bell Labs"
   homepage "http://graphviz.org/"
-  url "http://graphviz.org/pub/graphviz/stable/SOURCES/graphviz-2.38.0.tar.gz"
-  mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/g/graphviz/graphviz_2.38.0.orig.tar.gz"
-  sha256 "81aa238d9d4a010afa73a9d2a704fc3221c731e1e06577c2ab3496bdef67859e"
+
+  stable do
+    url "http://graphviz.org/pub/graphviz/stable/SOURCES/graphviz-2.38.0.tar.gz"
+    mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/g/graphviz/graphviz_2.38.0.orig.tar.gz"
+    sha256 "81aa238d9d4a010afa73a9d2a704fc3221c731e1e06577c2ab3496bdef67859e"
+
+    patch :p1, :DATA
```

Comment:
Unfortunately not. They originally just replaced the error codes (`e_VMerror` -> `gs_error_VMerror`), but that broke compatibility with older versions of the plugin, so *then* they switched the codes back, but added the `#ifndef` checks.

If GitHub supports a way to generate a diff for one file based on two compared commits, then we’ve got a shot, otherwise I think this is the best method.